### PR TITLE
Fix for SiCKRAGETV/sickrage-issues#2498: Include a print "Shows list" feature

### PR DESCRIPTION
**Pull request by MGaetan89**
Do not merge yet, as some points still need discussion in https://github.com/SiCKRAGETV/sickrage-issues/issues/2498.

Current status:
Print Shows List

### CODE REVIEW

File: gui/slick/interfaces/default/home_print.mako [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+%>
+
+<%
+    myDB = db.DBConnection()
+    today = str(datetime.date.today().toordinal())
+
+    status_quality  = '(' + ','.join([str(x) for x in Quality.SNATCHED + Quality.SNATCHED_PROPER]) + ')'
+    status_download = '(' + ','.join([str(x) for x in Quality.DOWNLOADED + [ARCHIVED]]) + ')'
+
+    sql_statement  = 'SELECT showid, '
```
Comment (MGaetan89) [inline gui/slick/interfaces/default/home_print.mako]: This query comes from `home.mako`.
I didn't have the courage to check if it does more than what is needed here.

**Comment by miigotu** [comment]
Closing, in favor of an @media print solution.

**Comment by MGaetan89** [comment]
There is no need to close the PR. I will update it.

**Comment by MGaetan89** [comment]
@miigotu What do you think I should do about images (poster/banner)? Should I hide them or not?

**Comment by miigotu** [comment]
@MGaetan89 I think not. If the user wants no posters printed they can use the simple layout when printing. And chrome has a checkbox in the print dialog "include images"

**Comment by MGaetan89** [comment]
I updated the PR to use the existing layouts and simply improve the print style.
The "Poster" layout still needs some :heart:, but other layouts are looking quite good.

What do you think about [hiding the URLs](https://github.com/SiCKRAGETV/SickRage/pull/2335/files#diff-8e81011888cc3a644cdf6f9cedc4adaaR8)? By default all URLs are displayed next to the link (so the poster URL and show URL). It takes a lot of space, so I hide them. If people want to see a show, they can directly access the web UI/mobile app, no?

**Comment by miigotu** [comment]
Yes, links/urls should be hidden. I will check this out and see what it looks like a bit later.

Is it ready for merge?

**Comment by MGaetan89** [comment]
I still want to try to improve the "Poster" layout. After that it will be ready to merge. Give me a few more days.

**Comment by MGaetan89** [comment]
I am out of idea to make the "Poster" layout properly fit in the page bounds. If you have ideas to improve it, let me know and I will include them.

Here is the result for each layout:

## Poster

Poster

## Small Poster

Small Poster

## Banner

Banner

## Simple

Simple

**Pull request merged**